src/components/Analytics/utils.ts

```diff
@@ -91,10 +90,17 @@ export enum EventAction {
   OPEN_TOOLTIP = 'open tooltip',
 }
 
+// Capitalize string (special characters are not kept)
 function toTitleCase(name: string) {
   return words(name).map(capitalize).join(' ');
 }
 
+// Capitalize string (special characters kept in place by not using "words" from lodash)
+function toTitleCaseWithSpecialCharacters(name: string) {
+  const words = name.split(' ');
+  return words.map(word => capitalize(word)).join(' ');
```

Comment:
You can simplify a little here (you don't need to create the arrow function inside `map`, see above [utils.ts#L95](https://github.com/covid-projections/covid-projections/pull/5909/files#diff-f0394b6ed9e1269459eda77202b48ae5bdd989fbd0fceab0b957487227204008R95)

```ts
return name.split(' ')
  .map(capitalize)
  .join(' ');
```